src/engine/enginebuffer.cpp

```diff
@@ -522,8 +522,10 @@ void EngineBuffer::loadFakeTrack(TrackPointer pTrack, bool bPlay) {
     if (bPlay) {
         m_playButton->set((double)bPlay);
     }
-    slotTrackLoaded(pTrack, pTrack->getSampleRate(),
-                    pTrack->getSampleRate() * pTrack->getDurationInt());
+    slotTrackLoaded(
+            pTrack,
+            pTrack->getSampleRate(),
+            pTrack->getSampleRate() * pTrack->getDurationSecondsInt());
```

Comment:
No change by this PR, but why we need to round to seconds, when we multiply by samples anyway? 
I think rounding after multiplication would be correct.

Reply:
Only used in tests. I don't wanto to modify any behavior.

Reply:
Can you add a TODO that we should investigate this?